src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -228,7 +227,6 @@ <h4 mat-dialog-title>Register Workflow</h4>
             [value]="descriptorType"
           >
             {{ descriptorType === 'gxformat2' ? 'Galaxy' : ('descriptor_type' | mapFriendlyValue: descriptorType) }}
```

Comment:
ah, why i didn't like `descriptorType === 'gxformat2' ? 'Galaxy' : ` being added, because i knew it's likely we'd forget to remove it